spec/api/holdings_api_spec.rb

```diff
@@ -232,8 +232,62 @@ def parse_body(call)
         build(:holding, organization: "umich", ocn: ocn),
         build(:holding, organization: "smu", ocn: ocn)
       )
-      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1, organization: "umich")))
+      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1)))
       expect(response["organizations"].sort).to eq ["smu", "umich"]
     end
   end
+
+  describe "v1/item_held_by_brlm" do
+    it "returns an application error if the item_id does not match an ht_item" do
+      response = parse_body(v1(url_template("item_held_by_brlm", item_id: item_id_1)))
+      expect(response["application_error"]).to eq "no matching data"
+    end
+    it "reports nothing if there are no holdings" do
+      load_test_data(htitem_1)
+      response = parse_body(v1(url_template("item_held_by_brlm", item_id: item_id_1)))
+      expect(response["organizations"]).to eq []
+    end
+    it "reports nothing if all holdings are status:CH & condition:''" do
```

Comment:
I hate to gripe about style (I'm usually fine with whatever standardrb does), but I find it makes it much easier to read the tests when the `it/end` blocks have a line of whitespace in between them...

Reply:
added vertival space between tests